shoop/admin/templates/shoop/admin/system/telemetry.jinja

```diff
@@ -25,7 +25,7 @@
                     {% if is_grace and opt_in %}
                         <p><i class="text-info fa fa-info-circle"></i> {% trans %}Telemetry submission is currently suspended; data will only be sent 24 hours after installation if still opted in at that point.{% endtrans %}</p>
                     {% elif last_submission_time %}
-                        <p>{% trans %}Telemetry data was last submitted at{% endtrans %} {{ last_submission_time|datetime }} - ({{ last_submission_time|timesince }} {% trans %}ago{% endtrans %})</p>
+                        <p>{{ _("Telemetry data was last submitted at %s - (%s) ago" % (last_submission_time|datetime, last_submission_time|timesince)) }}</p>
```

Comment:
This should probably just use a regular `{% trans %}` block?

Also, if you use the `_` function, you shouldn't use % interpolation; instead use named placeholders and kwargs, like

```jinja2
{{ _('Hello %(name)s!', name='World') }}
```